jobs/longitudinal_view.sh

```diff
@@ -5,14 +5,15 @@ if [[ -z "$bucket" || -z "$date" ]]; then
     exit 1
 fi
 
-git clone -b spark1.6 https://github.com/mozilla/telemetry-batch-view.git
+git clone -b spark2 https://github.com/mozilla/telemetry-batch-view.git
```

Comment:
Note that the job is temporarily using the `spark2` branch instead of `master` as the review for the required changes to telemetry-batch-view is still pending.